templates/layouts/standard/head/GoogleSearch.html

```diff
@@ -9,3 +6,7 @@
     <button type="submit" class="btn btn-icon-toggle ink-reaction"><i class="fa fa-search"></i></button>
 </form>
 </li>
+<link type="text/css" rel="stylesheet" href="/static/css/google-search.css" />
+<div class="hide-google">
+<gcse:searchresults-only></gcse:searchresults-only>
+</div>
```

Comment:
Good to [know](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).